index.js

```diff
@@ -74,19 +79,24 @@
 //. ```javascript
 //. const def = $.create ({
 //.   checkTypes: process.env.NODE_ENV === 'development',
-//.   env,
+//.   env: env,
 //. });
 //. ```
 //.
-//. `def` is a function for defining functions. For example:
+//. `def` is a function for defining functions.
+//.
+//# def :: String -> StrMap Array TypeClass -> Array Type -> Function
+//. -> Function
```

Comment:
I don't think we should use `//#` here, as `def` is not one of the module's exports. We could, though, include a type signature in the preceding code block:

```diff
 //. ```javascript
+//. //    def :: String -> StrMap (Array TypeClass) -> Array Type -> Function -> Function
 //. const def = $.create ({
 //.   checkTypes: process.env.NODE_ENV === 'development',
 //.   env,
 //. });
 //. ```
```

Reply:
updated.